src/index.html

```diff
@@ -7,8 +7,41 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+          class="image"
+          src="https://raw.githubusercontent.com/mate-academy/layout_moyo-header/master/src/images/logo.png"
+          alt=""
+        >
+      </a>
+      <nav class="nav">
+        <ul class="list">
+          <li class="row"><a href="#" class="link is-active">APPLE</a></li>
```

Comment:
```suggestion
          <li class="row">
            <a href="#" class="link is-active">APPLE</a>
          </li>
```
Новый тег - новая строка